### ISSUE TITLE
sort plugins alphabetically when listing

### ISSUE THREAD

**Issue by txoof** [txoof/PaperPi#97]
**Module/Plugin**
library/get_help

**Issue to be resolved**
Order the list of plugins alphabetically
``` text
--list_plugins                                       (I#93_sysinfo) X
11:21:58 paperpi :get_config_files:255 :INFO       - using configuration files to configure PaperPi: [PosixPath('/usr/local/paperpi/config/paperpi.ini'), PosixPath('/home/pi/.config/com.txoof.paperpi/paperpi.ini')]

get plugin information and user-facing functions:
Usage: --plugin_info PLUGIN_NAME|PLUGIN_NAME.FUNCTION
PLUGINS AVAILABLE:
  demo_plugin
  newyorker
  reddit_quote
  word_clock
  met_no
  default
  xkcd_comic
  librespot_client
  slideshow
  lms_client
  dec_binary_clock
  moon_phase
  crypto
  splash_screen
  basic_clock
```

**Line numbers/code references**
[lines](https://github.com/txoof/PaperPi/blob/34655e3206aa26c5ab60d2f84766e6141c2afd68/paperpi/library/get_help.py#L67-L80)
```python
def _get_modules(root='./plugins/'):
    '''get a list of modules contained within the path specified
    
    Args:
        root(str): path to search for modules
    
    Returns:
        list of str: list modules found within the root'''
    module_list = [] # wrap this in a sorted()
    p = Path(root).resolve()
    for i in p.glob('*'):
        if i.is_dir() and i.name[0] not in ('_', '.'):
            module_list.append(i.name)
    return module_list
```

**Supporting links/documentation**

**Comment by txoof** [comment]
this function should also check if there is a valid plugin `.py` file that matches the directory name to verify it exists.

**Comment by Harshal662** [comment]
def _get_modules(root='./plugins/'):
    '''get a list of modules contained within the path specified
    
    Args:
        root(str): path to search for modules
    
    Returns:
        list of str: list modules found within the root'''
    module_list = [] 
    p = Path(root).resolve()
    for i in p.glob('*'):
        if i.is_dir() and i.name[0] not in ('_', '.'):
            module_name = i.name
            module_file = p / module_name / (module_name + '.py')
            if module_file.exists() and module_file.is_file():
                module_list.append(module_name)
    return sorted(module_list)
    
We can use this

**Comment by txoof** [comment]
Exactly!

Can you pop that in a PR?

**Comment by Harshal662** [comment]
yeah sure

**Comment by tarunsamanta2k20** [comment]
I'd like to work on this

**Comment by txoof** [comment]
Closed with PR #99
